src/drivers/display/fb/lv_linux_fbdev.c

```diff
@@ -289,6 +328,11 @@ static void flush_cb(lv_display_t * disp, const lv_area_t * area, uint8_t * colo
         }
     }
 
+    /* Clean up temporary pixel buffer used for software rotation */
```

Comment:
Can we pre-allocate memory and avoid frequent temporary releases and requests for memory?

Reply:
That's a good point. I modified the code to reuse the buffer across flushes and only re-allocate it when the display size changed.